03-task-manager/starter/app.js

```diff
@@ -1,17 +1,17 @@
 const express = require("express");
 const app = express();
 const mongoose = require("mongoose");
+const notFount = require("./middleware/not-found");
+const errorHandler = require("./middleware/error-handler");
 require("dotenv").config();
 
 const taskRoutes = require("./routes/tasks");
 
 app.use(express.json());
 
 app.use("/api/v1/tasks", taskRoutes);
-
-app.all("*", (req, res) => {
-  res.status(404).send("Page Not Found!");
-});
+app.use(errorHandler);
```

Comment:
If I'm not mistaken, your `errorHandler` function should be the last of your middleware